modules/consul-cluster/outputs.tf

```diff
@@ -39,17 +39,17 @@ output "firewall_rule_intracluster_name" {
 }
 
 output "firewall_rule_inbound_http_url" {
-  value = "${google_compute_firewall.allow_inboud_http_api.self_link}"
+  value = "${google_compute_firewall.allow_inbound_http_api.*.self_link}"
 }
 
 output "firewall_rule_inbound_http_name" {
-  value = "${google_compute_firewall.allow_inboud_http_api.name}"
+  value = "${google_compute_firewall.allow_inbound_http_api.*.name}"
```

Comment:
This will probably fail when the `count` on `google_compute_firewall.allow_inbound_http_api` is set to 0. See https://github.com/hashicorp/terraform/issues/16726. You'll need to update this to something like:

```hcl
+  value = "${element(concat(google_compute_firewall.allow_inbound_http_api.*.name, list(""))}"
```

This applies to all the other `.*.` output values.